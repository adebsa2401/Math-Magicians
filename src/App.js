import React from 'react';
import './App.css';
import Calculator from './components/Calculator';

export default class App extends React.PureComponent {
  render() {
    return (
      <Calculator />
    );
  }
}
