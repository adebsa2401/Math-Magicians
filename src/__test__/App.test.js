import renderer from 'react-test-renderer';
import App from '../App';

it('The App component renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
