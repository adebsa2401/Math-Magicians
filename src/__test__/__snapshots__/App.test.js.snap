// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App component renders correctly 1`] = `
Array [
  <header>
    <h1>
      Math Magicians
    </h1>
    <nav>
      <a
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="#/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="#/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </nav>
  </header>,
  <main>
    <div
      className="home-main"
    >
      <h2>
        Welcome to our page!
      </h2>
      <p>
        Ea amet enim minim Lorem exercitation incididunt. Mollit consequat deserunt amet enim labore enim cupidatat irure nulla nisi. Commodo esse in exercitation id aliquip commodo sit cupidatat ullamco enim nisi voluptate eiusmod cillum. Qui sunt ex fugiat id quis cupidatat elit tempor. Non sit amet enim sint ullamco officia culpa Lorem magna culpa nostrud.
      </p>
      <p>
        Est ipsum deserunt amet quis do quis incididunt consectetur aliqua ea dolor exercitation. Duis sint aliquip velit quis. Adipisicing elit aliquip ipsum officia do tempor irure mollit amet anim labore.
      </p>
    </div>
  </main>,
]
`;
